doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -0,0 +1,90 @@
+.. _thread_ot_memory:
+
+OpenThread memory requirements
+##############################
+
+This page provides information about the layout and the amount of Flash and RAM memory required by the sample using the OpenThread stack.
+These values are useful to see whether your application has enough space for running on paricular platforms in different OpenThread configurations.
+
+
+Flash and RAM requirements for nRF52840 and nRF52833
+****************************************************
+
+The following tables list flash memory and RAM requirements for the provided samples built using the OpenThread libraries.
+The values change depending on the application and hardware platform.
+
+The values in the tables are provided for the CLI example, which works with all OpenThread calls.
+For this reason, these values are the best approximation of the memory requirements.
```

Comment:
This is a 1:1 copy from nRF5 SDK for TZ. Please update to NCS.
Also, does this mean the CLI sample has the highest memory requirements? Because in line 13 you say these values are valid for all samples.

Reply:
`Values for the :ref:`Thread CLI sample <ot_cli_sample>`, which works with all OpenThread calls, are only an approximation of the real memory requirements for a CLI application.`

No, this is a real CLI application, so measured requirements **are showing** real memory requirements.